wrp.go

```diff
@@ -53,6 +53,7 @@ var (
 	fgeom       = flag.String("g", "1152x600x216", "Geometry: width x height x colors, height can be 0 for unlimited")
 	htmFnam     = flag.String("ui", "wrp.html", "HTML template file for the UI")
 	delay       = flag.Duration("s", 2*time.Second, "Delay/sleep after page is rendered and before screenshot is taken")
+	token       = flag.String("token", "", "If set, all requests need to have this set as Bearer header")
```

Comment:
Guess this goes a bit the go idiom of keeping names short, but tbh - i don't care since `t` is already used.